21-inferential-analysis.Rmd

```diff
@@ -289,19 +289,19 @@ The results are the same and, based on these and the confidence interval for thi
 
 ## A More Complex Model
 
-For count data, there are occasions where the number of zero counts is larger than what a simple Poisson distribution would prescribe. A more complex model appropriate in this situation is the zero-inflated Poisson (ZIP) model; see @Mullahy, @Lambert1992, and @JSSv027i08. Here, there are two sets of covariates: one for the non-zero count data and others that affect the probability of zeros (denoted as $\pi$). The mean function is
+For count data, there are occasions where the number of zero counts is larger than what a simple Poisson distribution would prescribe. A more complex model appropriate in this situation is the zero-inflated Poisson (ZIP) model; see @Mullahy, @Lambert1992, and @JSSv027i08. Here, there are two sets of covariates: one for the count data and others that affect the probability of zeros (denoted as $\pi$). The mean function is
 
-$$\lambda = 0 \pi + (1 - \pi) \lambda_{nz}$$
```

Comment:
@topepo can you double-check this equation especially and this little section overall? Dan had some uncertainty about it, and I think we just need to take out the zero??

Reply:
That is the correct equation (this has come up before). That's how the original authors write it. It may look weird but it is correct. 

I also think that "non-zero count" verbiage should stay.